sources/north-america/us/oh/ODOT_TIMS_Road_Inventory.geojson

```diff
@@ -0,0 +1,1493 @@
+{
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "type": "Feature",
+      "properties": {
+        "id": "ODOT_TIMS_Road_Inventory",
+        "name": "ODOT (Ohio) Road Inventory",
+        "description": "Ohio Department of Transportation (ODOT) - Transportation Information Mapping System (TIMS) Road Inventory published June 2024.",
+        "type": "tms",
+        "url": "https://tiles.mblaine.com/ohio/{zoom}/{x}/{y}.png",
+        "min_zoom": 0,
+        "max_zoom": 19,
+        "license_url": "https://wiki.openstreetmap.org/wiki/ODOT_TIMS#Right_to_Use",
+        "privacy_policy_url": "https://www.transportation.ohio.gov/help-center/privacy-notice-and-policies",
+        "permission_osm": "implicit",
+        "icon": "https://dam.assets.ohio.gov/image/upload/transportation.ohio.gov/assets/Logo_Assets/ODOT_Zephyr-HORZ-RGB.png",
+        "i18n": true,
+        "country_code": "US",
+        "attribution": {
+          "url": "https://gis.dot.state.oh.us/tims_classic/About",
+          "text": "Ohio Department of Transportation",
+          "required": false
+        },
+        "end_date": "2024-02-09",

```

Comment:
How often do you plan to update this layer with respect to TIMS? If you plan to do it several times a year, I’m guessing we should omit the end date so it doesn’t get stale. ELI updates only make it into iD with each release of iD, which can be several months apart.

Reply:
ODOT says they release their updates annually. So this year and apparently last year the data was published the last week of July. If the next published update is July 2025, does it make sense to include end_date?

Reply:
Sure, just remember to update the timestamp whenever you update the tileset.

Reply:
Will do!